guides/common/modules/con_salt-introduction.adoc

```diff
@@ -1,7 +1,7 @@
 [id="salt_guide_introduction_{context}"]
 = Introduction to Salt
 
-Salt can be used as a configuration management tool similar to {ConfiguringAnsibleDocURL}using-ansible-roles_ansible[Ansible] or {ManagingHostsDocURL}using-puppet-for-configuration-management_managing-hosts[Puppet].
+Salt can be used as a configuration management tool similar to {ConfiguringAnsibleDocURL}using-ansible-roles_ansible[Ansible] or {ManagingConfigurationsPuppetDocURL}[Puppet].
```

Comment:
I'd suggest switching the wording around here so that it's active and user oriented instead of passive. That'd be compliant with the IBM style guide and easier for translation too. So something like:

You can use Salt as a configuration management tool similar to {ConfiguringAnsibleDocURL}using-ansible-roles_ansible[Ansible] or {ManagingHostsDocURL}using-puppet-for-configuration-management_managing-hosts[Puppet].